diesel/src/query_builder/only_clause.rs

```diff
@@ -0,0 +1,85 @@
+//! `ONLY` clause.
+//!
+//! Can only be used on tables, e.g. `my_table::dsl::table.only().select(something)` should generate
+//! something like `SELECT something FROM ONLY my_table`.
+
+use super::{AsQuery, AstPass, Backend, QueryFragment, QueryId, SelectStatement};
+use crate::{
+    associations::HasTable, expression::ValidGrouping, prelude::*, query_dsl::methods::*,
+    query_source::Table,
+};
+
+/// Represents a query with an `ONLY` clause.
+#[derive(Debug, QueryId)]
+pub struct Only<T> {
+    query: T,
+}
+
+/// Add `ONLY` before a query source.
+pub trait OnlyDsl: AsQuery + Sized {
```

Comment:
The dsl should be moved to the querydsl module (or the corresponding submodule of the pg module)

Reply:
I can't find any dsl module in `src/pg`

Reply:
I would likely add it [here](https://github.com/diesel-rs/diesel/tree/master/diesel/src/pg/expression/extensions) as additional sub module + reexport it from `diesel::query_dsl`

Reply:
I guess it could go directly into the `pg` module?